packages/devtools_app/lib/src/screens/inspector_v2/inspector_data_models.dart

```diff
@@ -246,7 +246,7 @@ class LayoutProperties {
     if (parentElement == null) return this;
     final parentProperties =
         parentElement.computeLayoutProperties(forFlexLayout: false);
-    return parentProperties;
+    return parentProperties ?? this;
```

Comment:
Does this return type still need to be nullable?

Reply:
Good point, no it doesn't!

Reply:
Ah actually it needs to still be nullable because a subclass which overrides it returns null.